buildSrc/build.gradle

```diff
@@ -45,7 +60,7 @@ dependencies {
     compile 'org.springframework.build.gradle:propdeps-plugin:0.0.7'
 
     if (localRepo) {
-        compile name: "build-tools-${version}"
+        compile name: "build-tools-${esVersion}"
     } else {
         compile group: 'org.elasticsearch.gradle', name: 'build-tools', version: esVersion
```

Comment:
This forces the qualified ES to exist in the remote repo prior to allowing this build to succeed... is that intentional ?
For example
```
./gradlew clean assemble -Dbuild.version_qualifier=alpha1 -Dbuild.snapshot=false
```
will not build until ES alpha1 (not snapshot) is available on the remote repo, in the prior version, it was controlled solely by the property which, (to me) is unclear if this is a change in the behavior/workflow for the release.

Reply:
Further up in the script the `localRepo` variable is set from the system properties. The localRepo property is used to configure a local directory as a repository for resolving project dependencies. During the release process, Elasticsearch is built first and the resulting build tools artifact is copied to the localRepo directory in the ES-Hadoop project. This way ES-Hadoop can depend on the exact same artifact that will be released by the same build. Those setup steps and scripts are managed outside this script by the release automation.

Reply:
Thanks for the explains!